DependencyInjection/Configuration.php

```diff
@@ -78,6 +79,37 @@ public function getConfigTreeBuilder()
             ->end()
         ;
 
+        $this->addAlternateSection($rootNode);
+
         return $treeBuilder;
     }
+
+    private function addAlternateSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('alternate')
+                    ->info('Section can be enabled to generate alternate (hreflang) urls')
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('default_locale')
+                            ->defaultNull()
+                            ->info('The default locale used by url loc')
+                        ->end()
+                        ->arrayNode('locales')
+                            ->beforeNormalization()
+                            ->ifString()
+                                ->then(function ($v) { return preg_split('/\s*,\s*/', $v); })
+                            ->end()
+                            ->prototype('scalar')->end()
+                            ->info('Array of locales to generate alternate (hreflang) urls')
+                        ->end()
+                        ->enumNode('i18n')
+                            ->defaultValue('symfony')
+                            ->values(['symfony', 'jms'])
+                            ->info('Name of project bundle to create i18n routes. Possible values are symfony or jms')
+                    ->end()
```

Comment:
We may be missing an `end()` (or the indentation is wrong)

Reply:
Thanks, my mistake